README.md

```diff
@@ -341,8 +341,56 @@ LIT_SERVER_API_KEY=supersecretkey python main.py
 Clients are expected to auth with the same API key set in the `X-API-Key` HTTP header.
 
 </details>
+
+
+<details>
+  <summary>Stream long responses</summary>
+
+&nbsp;
+
+LitServe can stream outputs from the model in real-time, such as returning text one word at a time from a language model.
+
+To enable streaming, you need to set `LitServer(..., stream=True)` and  implement `LitAPI.predict` and `LitAPI.encode_response`
+as a generator (a Python function that yields output).
+
+For example, streaming long responses generated over time:
+
+```python
+from typing import Generator
+
+import json
+from litserve import Request
+from litserve import LitServer
+from litserve import LitAPI
```

Comment:
in examples you MUST minimize the number of import lines...

as a brand we want to do:

```
import litserve as ls
```

which removes 3 import lines because now you can do:
```
ls.Request
ls.LitServer
ls.LitAPI
```

and the top imports can be made a single line:

```
import typing.Generator as Generator, json
```

the result is:
```
import typing.Generator as Generator, json
import litserve as ls
```

Reply:
thank you! applied the feedback. I also removed `Generator` type hint since that's not really required to understand what is happening during streaming.